fetch_submissions.py

```diff
@@ -40,19 +22,38 @@ def progress(count, total, status=''):
 
 
 def get_file_extension(file_type):
-  if file_type in FILE_EXTENSIONS:
-    return FILE_EXTENSIONS[file_type]
-  return 'txt'
-
-
-def get_comment_symbol(file_type):
-  if file_type in ['sql']:
-    return '--'
-  if file_type in ['python3', 'python', 'php', 'ruby', 'bash']:
-    return '#'
-  if file_type in ['csharp', 'rust', 'kotlin', 'scala', 'golang', 'swift',
-                   'javascript', 'c', 'java', 'cpp']:
-    return '//'
+  try:
+    return {
+      'mysql': 'sql',
+      'bash': 'sh',
+      'python3': 'py',
+      'php': 'php',
+      'rust': 'rs',
+      'kotlin': 'kt',
+      'scala': 'scala',
+      'golang': 'go',
+      'swift': 'swift',
+      'ruby': 'rb',
+      'javascript': 'py',
+      'csharp': 'cs',
+      'c': 'c',
+      'python': 'py',
+      'java': 'java',
+      'cpp': 'cpp'
+    }[file_type]
+  except KeyError:
+    return 'txt'
+
+
+def get_comment_symbol(lang):
+  lang_comment_symbol_mapping = {
+      'mysql': '--',
+      ('python3', 'python', 'php', 'ruby', 'bash'): '#',
+      ('csharp', 'rust', 'kotlin', 'scala', 'golang', 'swift', 'javascript', 'c', 'java', 'cpp'): '//'
+  }
+  for langs in lang_comment_symbol_mapping.keys():
+      if lang in langs:
+          return lang_comment_symbol_mapping[langs]
```

Comment:
I've grouped the languages based on their comment symbol and created a mapping from the group to the symbol. This way checking for the symbol can be done using the dictionary. If the language is not found (exceptional case given that all languages supported on LeetCode are mentioned in the script) then we return plaintext comments in the code text files.

Reply:
What is your rationale for having this? I fail to see how this is any more readable or efficient than the current implementation.

Reply:
It's definitely no more efficient than the current implementation. Replacing multiple if statements with a dictionary was my personal preference. This change is more consistent with the one I made for the `get_file_extension` function. If you feel that it is not any more readable then I can remove that changes.

Reply:
Yeah, I feel like constructing the dictionary does not add significantly to this part of the script :/